src/gtimelog/main.py

```diff
@@ -74,7 +67,7 @@ def mark_time(what=None):
 
 mark_time("Gtk imports done")
 
-from gtimelog import __version__
+from gtimelog import DEBUG, __version__
```

Comment:
DEBUG was already imported.  I'm surprised flake8 didn't complain.

Do I have flake8 in my Travis CI pipeline?  Yes, I do.  Strange.